### PR TITLE
TASK#36837 - Added ReadOnly prop checkbox for textarea, select, radio…

### PR DESCRIPTION
**Pull request by SebastianPietruszka**
…, checkbox, number stories in alto storybook

### CODE REVIEW

File: src/alto-ui/Form/CheckBox/story.js [hunk 1]
```diff
@@ -2,15 +2,16 @@
...
 storiesOf('Form/CheckBox', module)
   .addDecorator(centered)
   .addWithJSX('overview', () => (
     <div>
-      <CheckBox id="unchecked" label="Unchecked" />
-      <CheckBox id="checked" label="Checked" checked />
+      <CheckBox id="unchecked" label="Unchecked" readOnly={boolean('readOnly', false)} />
+      <CheckBox id="checked" label="Checked" defaultChecked readOnly={boolean('readOnly', false)} />
       <CheckBox id="disabled" label="Disabled" disabled />
       <CheckBox id="disabled-checked" label="Disabled checked" disabled checked />
```
Comment (kevinmlsilva) [inline src/alto-ui/Form/CheckBox/story.js]: You should create two new checkboxes. so on the storybook we can test the ones without readOnly.
on checked and other not

File: src/alto-ui/Form/Radio/story.js [hunk 2]
```diff
@@ -2,15 +2,22 @@
...
+import { boolean } from '@storybook/addon-knobs';
 
 import Radio from './Radio';
 
 storiesOf('Form/Radio', module)
   .addDecorator(centered)
   .addWithJSX('overview', () => (
     <div>
-      <Radio id="inactive" label="Inactive" name="overview" />
-      <Radio id="checked" label="Checked" name="overview" checked />
```
Comment (kevinmlsilva) [inline src/alto-ui/Form/Radio/story.js]: You should create two new radios. so on the storybook we can test the ones without readOnly.
on checked and other not

**Comment by SebastianPietruszka** [comment]
I also change `checked` to `defaultChecked` to avoid this warning: 
warning

File: src/alto-ui/Form/Radio/story.js (hunk 2 above)
Comment (SebastianPietruszka) [reply src/alto-ui/Form/Radio/story.js]: Fixed

File: src/alto-ui/Form/CheckBox/story.js (hunk 1 above)
Comment (SebastianPietruszka) [reply src/alto-ui/Form/CheckBox/story.js]: Fixed

File: src/alto-ui/Form/CheckBox/CheckBox.js [hunk 3]
```diff
@@ -32,6 +32,7 @@ CheckBox.propTypes = {
   disabled: PropTypes.bool,
   hideLabel: PropTypes.bool,
   readOnly: PropTypes.bool,
+  defaultChecked: PropTypes.bool,
```
Comment (renaudtertrais) [inline src/alto-ui/Form/CheckBox/CheckBox.js]: you are spreading props so not needed. You should have an eslint warning btw.

Comment (SebastianPietruszka) [reply src/alto-ui/Form/CheckBox/CheckBox.js]: Ok, fixed

**Pull request merged**